test/Microsoft.Azure.Devices.Edge.Test.Common/OsPlatform.cs

```diff
@@ -57,13 +57,9 @@ protected void InstallTrustedCertificates(IEnumerable<X509Certificate2> certs, S
             (string name, string args) command,
             CancellationToken token)
         {
-            await Profiler.Run(
-                async () =>
-                {
-                    string[] output = await Process.RunAsync(command.name, command.args, token);
-                    Log.Verbose(string.Join("\n", output));
-                },
-                "Executed: " + command.name + ' ' + command.args );
+            Log.Verbose("Executed: " + command.name + ' ' + command.args);
```

Comment:
Since this Log call happens before the command is executed, "Executed" should be "Execute" or "Executing"...

Reply:
EnGiRsh Is HaRd. 
`Executed` -> `Executing`